OH-Archive/_layouts/post.html

```diff
@@ -112,24 +113,12 @@ <h2 style="text-align: center">{{content}}
                 <div class="row" style="padding: 15px">
                     <h1> Weitere Informationen zum Interview</h1>
                     <div class="col">
-                        {% if page.birthday != "ubk" %}
```

Comment:
good, but as you can see in the .md-files: The data in question is still in there. Please remove id from the posts pages aswell.

To do this: adjust the md-generation here: `OH-Archive/backend/generate-md-files.js`

remove line 72: `                    + "\nlocation: " + geo.hasGeoloc`
and line 71: `                    + "\ncreator: " + general.creator`

Reply:
You mean line 72 and 73? : 
                   ```
 + "\nbirthday: " + person.birthday.toString().replace(/-/g, ".")
                    + "\nlocation: " + geo.hasGeoloc
```
since general.creator is used in line 125 in post.html : 

` <h2>Aufgenommen am {{page.interviewDate}} von {{page.creator}}</h2>`

I'll wait for you to confirm this before doing any changes

Reply:
also another thing, since we are no longer in need of these elements, should we also remove them from the "Entry" class in indexScript.js?

Reply:
yers, this is a very good remark! I'll do that!